permission_handler_windows/windows/permission_constants.h

```diff
@@ -24,28 +24,29 @@ class PermissionConstants {
         PHOTOS_ADD_ONLY = 10,
         REMINDERS = 11,
         SENSORS = 12,
-        SMS = 13,
-        SPEECH = 14,
-        STORAGE = 15,
-        IGNORE_BATTERY_OPTIMIZATIONS = 16,
-        NOTIFICATION = 17,
-        ACCESS_MEDIA_LOCATION = 18,
-        ACTIVITY_RECOGNITION = 19,
-        UNKNOWN = 20,
-        BLUETOOTH = 21,
-        MANAGE_EXTERNAL_STORAGE = 22,
-        SYSTEM_ALERT_WINDOW = 23,
-        REQUEST_INSTALL_PACKAGES = 24,
-        APP_TRACK_TRANSPARENCY = 25,
-        CRITICAL_ALERTS = 26,
-        ACCESS_NOTIFICATION_POLICY = 27,
-        BLUETOOTH_SCAN = 28,
-        BLUETOOTH_ADVERTISE = 29,
-        BLUETOOTH_CONNECT = 30,
-        NEARBY_WIFI_DEVICES = 31,
-        VIDEOS = 32,
-        AUDIO = 33,
-        SCHEDULE_EXACT_ALARM = 34
+        SENSORS_ALWAYS = 13,

```

Comment:
Same here, revert the reordering and add the `SENSOR_ALWAYS` definition at the bottom of the list.